### PR TITLE
Add JsonPath queries to README.md file.

### PR DESCRIPTION
**Pull request by roimenashe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,15 @@ Consider the following json
 }
 ```
 
+### Creating an Aerospike Document Client
+
+The Aerospike Document Client is instantiated as follows
+
+``` java
+   AerospikeClient client = new AerospikeClient(AEROSPIKE_SERVER_IP, AEROSPIKE_SERVER_PORT);
```
Comment (reugn) [inline README.md:67]: Mention in a comment that there are other constructors for AerospikeClient.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,15 @@ Consider the following json
...
+   AerospikeDocumentClient documentClient = new AerospikeDocumentClient(client)
```
Comment (reugn) [inline README.md:68]: Semicolon.

File: README.md [hunk 3]
```diff
@@ -113,115 +122,92 @@ We can delete a node e.g. the Medium reviewer's rankings
    documentClient.delete(tommyLeeJonesDBKey,"$.best_films_ranked[1]");
 ```
 
-### Document API
+### JsonPath Queries
 
-Below is the interface against which the API has been written
+JsonPath queries is java DSL for reading JSON documents.
```
Comment (reugn) [inline README.md:127]: [JsonPath](https://goessner.net/articles/JsonPath/) is a query language for JSON.

Comment (roimenashe) [reply README.md:127]: I took it directly from the official JsonPath repo:
https://github.com/json-path/JsonPath

File: README.md (hunk 1 above)
Comment (roimenashe) [reply README.md:67]: There will be only 1 after the next PR (documentBinName redesign).

Comment (reugn) [reply README.md:67]: I meant [this](https://docs.aerospike.com/apidocs/java/com/aerospike/client/AerospikeClient.html) one.

File: README.md (hunk 3 above)
Comment (reugn) [reply README.md:127]: I think we can omit the details of the underlying Java library that we are using to fill in the functionality gaps, but rather introduce the user to the JsonPath specification.

**Comment by roimenashe** [comment]
I addressed all comments, please review again.

**Review by reugn — APPROVED** [review]

**Pull request merged**